lib/reddex_web/views/layout_view.ex

```diff
@@ -1,3 +1,7 @@
 defmodule ReddexWeb.LayoutView do
   use ReddexWeb, :view
+
+  def signed_in?(conn) do
+    Plug.Conn.get_session(conn, :current_user)
```

Comment:
`!!Plug.Conn.get_session(conn, :current_user)` ?

I'd expect the `signed_in?` function to return `true/false`.